SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -817,5 +823,34 @@ private void CreatureManager()
                     Gui.TogglePlayerWindow(user, GuiWindowType.CreatureManager);
                 });
         }
+
+        private void DMBroadcast()
+        {
+            _builder.Create("bc")
+                .Description("Sends your DM shout to Discord.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .Action((user, target, location, args) =>
+                {
+                    var dmMessage = string.Join(" ", args);
+                    var url = Environment.GetEnvironmentVariable("SWLOR_DM_SHOUT_WEBHOOK_URL");
+
+                        for (var onlinePlayer = GetFirstPC(); GetIsObjectValid(onlinePlayer); onlinePlayer = GetNextPC())
```

Comment:
This line and 838 are over to the right by one extra tab. Please remove one tab from both of these lines so it lines up with the rest of the file.